generators/app/templates/infrastructure/package.json

```diff
@@ -60,6 +60,7 @@
     "graphql-subscriptions": "^1.2.1",
```

Comment:
I think we should have only one if with addSubscriptions: <%_ if(addSubscriptions)...